sbin/setkey/setkey.8

```diff
@@ -326,17 +330,16 @@ Do not allow cyclic sequence number.
 .It Fl lh Ar time
 .It Fl ls Ar time
 Specify hard/soft life time duration of the SA.
-.It Fl natt Ar oai \([ Ar sport \(] Ar oar \([ Ar dport \(]
+.It Fl natt Ar oai \(lB Ar sport \(rB Ar oar \(lB Ar dport \(rB
```

Comment:
Could you elaborate on this?  At first glance, it looks like this line produces the same output, but I find `\([` easier to understand than `\(lB`.

If you were replacing `\([` with `Po`, I'd understand the change -- moving from low-level groff to higher-level mdoc format.  But as it is, I don't see the benefit?

Reply:
Happily! `\([` is not real roff(7) escape sequence. The correct escape sequence for right bracket is `\([`. `Po` produces parenthesis, and `Oo` is for optional elements, of which this is not, the `[` is actually part of the syntax.

Reply:
We could use Bo and Bc?